requirements.txt

```diff
@@ -12,3 +12,7 @@ xlrd
 openpyxl
 metaflow_extensions@git+https://github.com/nestauk/metaflow_extensions@25_fix_project_env
 pyyaml
+requests-cache
+selenium
+requests
+WebDriver
```

Comment:
`WebDriver` is a class name and not a Python package listed on PyPi. See my comment above for how you could resolve the issue around web driver installation.